sklearn/model_selection/tests/test_split.py

```diff
@@ -581,6 +581,25 @@ def test_shuffle_split_default_test_size(split_class, train_size, exp_train,
     assert len(X_test) == exp_test
 
 
+@pytest.mark.parametrize("train_size, exp_train, exp_test",
+                         [(None, 8, 2),
+                          (8, 8, 2),
+                          (0.8, 8, 2)])
```

Comment:
Please add a case where `train_size=0.7` for instance: in this case `test_size=None` (the default) should be interpreted as the complement and therefore `exp_train=7` and `exp_test=3`.